components/Footer/src/index.scss

```diff
@@ -0,0 +1,95 @@
+.denhaag-footer {
+  --utrecht-paragraph-color: #fff;
+  --utrecht-paragraph-font-size: 16px;
+  --utrecht-paragraph-line-height: normal;
+  --utrecht-paragraph-margin-block-start: 12px;
+  --utrecht-paragraph-margin-block-end: 12px;
+
+  background-color: #4b4b4b;
+
+  --utrecht-link-social-background-color: transparent;
+  --utrecht-link-social-border-color: transparent;
+  --utrecht-link-social-border-width: 5px;
+  --utrecht-list-social-item-margin-inline-end: 20px;
+  --utrecht-link-social-color: #fff;
+  --utrecht-link-social-hover-color: var(--denhaag-color-ocher-2);
+
+  .utrecht-list-social {
+    align-items: baseline;
+  }
+
+  .denhaag-footer__mega {
+    background-color: var(--denhaag-color-grey-5);
```

Comment:
I would set the `denhaag.footer.mega.background-color` to `{denhaag.color.grey.5}` in JSON and get it here with `var(--denhaag-footer-mega-background-color)` 

For A11y tests later it might also be helpful to always set a text color too. So `denhaag-footer-mega-color` :)